README.md

```diff
@@ -30,48 +30,78 @@
+- `Highly Modular Framework.` OmniSafe presents a highly modular framework, incorporating an extensive collection of tens of algorithms tailored for safe reinforcement learning across diverse domains. The framework achieves versatility through the utilization of algorithm-level abstraction and API interfaces, employing two distinctive design components: Adapter and Wrapper.
+
+- `High-performance parallel computing acceleration.` OmniSafe not only supports environment-level asynchronous parallelism but also incorporates agent asynchronous learning. By harnessing the capabilities of Torch.distributed, OmniSafe accelerates the learning process of algorithms by creating unique agents within individual threads.
+
+- `Out-of-box toolkits.` OmniSafe offers customizable toolkits for tasks like training, benchmarking, analyzing, and rendering. [Tutorials](https://github.com/OmniSafeAI/omnisafe#getting-started) and user-friendly [APIs](https://omnisafe.readthedocs.io/en/latest/baserlapi/on_policy.html) make it easy for beginners and average users, while advanced researchers can enhance their efficiency without complex code.
+
+![Train video](./images/train.gif)
 
 --------------------------------------------------------------------------------
 
 ### Table of Contents  <!-- omit in toc --> <!-- markdownlint-disable heading-increment -->
-
-- [Implemented Algorithms](#implemented-algorithms)
-  - [Latest SafeRL Papers](#latest-saferl-papers)
-  - [List of Algorithms](#list-of-algorithms)
-    - [On-Policy Safe](#on-policy-safe)
-    - [Off-Policy Safe](#off-policy-safe)
-    - [Model-Based Safe](#model-based-safe)
-    - [Offline Safe](#offline-safe)
-    - [Others](#others)
-- [Installation](#installation)
-  - [Prerequisites](#prerequisites)
-    - [Install from source](#install-from-source)
-    - [Install from PyPI](#install-from-pypi)
-  - [Examples](#examples)
-    - [Try with CLI](#try-with-cli)
-- [Getting Started](#getting-started)
-  - [Important Hints](#important-hints)
-  - [Quickstart: Colab on the Cloud](#quickstart-colab-on-the-cloud)
-- [Changelog](#changelog)
-- [The OmniSafe Team](#the-omnisafe-team)
-- [License](#license)
+- [🚀Quick Start](#quick-start)
+  - [🔥Installation](#installation)
+    - [👀Prerequisites](#prerequisites)
+    - [✨Install from source](#install-from-source)
+    - [✨Install from PyPI](#install-from-pypi)
+- [🍪Implemented Algorithms](#implemented-algorithms)
+  - [🍩Examples](#examples)
+    - [🏠Algorithms Registry](#algorithms-registry)
+    - [☁️Supported Environments](#️supported-environments)
+    - [🚢Try with CLI](#try-with-cli)
+- [🍷Getting Started](#getting-started)
+  - [🥢Important Hints](#important-hints)
+  - [🍴Quickstart: Colab on the Cloud](#quickstart-colab-on-the-cloud)
+- [🔖Changelog](#changelog)
+- [⚓The OmniSafe Team](#the-omnisafe-team)
+- [📖License](#license)
```

Comment:
I still think we should not add emojis to section headers.

Reply:
Why